crates/transport_iroh/src/lib.rs

```diff
@@ -319,11 +345,7 @@ impl IrohTransport {
             return Err(err);
         }
 
-        // Send peer URL to remote once per connection.
-        if let Some(local_url) = local_url {
-            let payload = Bytes::copy_from_slice(local_url.as_str().as_bytes());
-            send_frame(&conn, FrameType::PeerUrl, payload).await?;
-        }
+        tokio::time::sleep(Duration::from_millis(1000)).await;
```

Comment:
A TODO here would be good too.

Reply:
https://github.com/holochain/kitsune2/pull/394/commits/7a85080f1f80c3d696c89648499d0eb2fbdac9a9